app/components/views/GetStartedPage/WalletSelection/Form.js

```diff
@@ -1,65 +1,78 @@
 import CreateWalletForm from "./CreateWalletForm";
-import SelectAvailableWalletsForm from "./SelectAvailableWalletsForm";
 import { FormattedMessage as T, injectIntl } from "react-intl";
-import { KeyBlueButton, RemoveWalletButton } from "buttons";
+import { KeyBlueButton, RemoveWalletButton, InvisibleButton } from "buttons";
 import "style/LoginForm.less";
 
 const WalletSelectionBodyBase = ({
   availableWallets,
-  sideActive,
-  onShowCreateWallet,
-  onShowSelectWallet,
   createWallet,
+  showCreateWalletForm,
+  hideCreateWalletForm,
+  createWalletForm,
   startWallet,
   onRemoveWallet,
   selectedWallet,
+  onChangeAvailableWallets,
   intl,
   ...props,
 }) => {
   return (
-    availableWallets && availableWallets.length > 0 ?
-      <Aux>
-        <div className="advanced-page-toggle">
-          <div className="text-toggle">
-            <div className={"text-toggle-button-left " + (!sideActive && "text-toggle-button-active")} onClick={sideActive ? onShowCreateWallet : null}>
-              <T id="advancedDaemon.toggle.remote" m="Select Available Wallet" />
-            </div>
-            <div className={"text-toggle-button-right " + (sideActive && "text-toggle-button-active")} onClick={!sideActive ? onShowSelectWallet : null}>
-              <T id="advancedDaemon.toggle.appdata" m="Create New Wallet" />
-            </div>
+    availableWallets && availableWallets.length > 0 && !createWalletForm ?
+      <div className="advanced-page">
+        <div className="advanced-page-form">
+          <div className="advanced-daemon-row">
+            {availableWallets.map(wallet => {
+              const selected = wallet.value.wallet == selectedWallet.value.wallet;
```

Comment:
This is making the default-wallet of both mainnet and testnet get selected, and then when starting it seems only the mainnet wallet is getting started.

Reply:
I believe you need to check by the wallet.label, which has the network appended

Reply:
I am having the same issue. Here is what the log says: 
```
2018-03-05 09:41:52.018 [ERR] DCTN: Start RPC errored  {"code":5,"metadata":{"_internal_repr":{}},"details":"Connection to RPC server failed: mismatched networks"} 
```